.gitignore

```diff
@@ -0,0 +1,3 @@
+venv/
+User/ microservice/__pycache__/
+User/ microservice/config.ini
```

Comment:
we need to make sure that these file are already git-ignored,

because I saw `__pycache__` and `config.ini` files already pushed.

we need to delete them.